_posts/randy/2022-07-03-all-old-movies-are-garbage.md

```diff
@@ -0,0 +1,95 @@
+image: "assets/img/old-movies-are-garbage.png"
+description: Mine 
+tags: tails analog-sampling
+---
+
+![Old Movies Are Garbage](/assets/img/old-movies-are-garbage.png)
+
+>video width="720" height="480" controls>
+   >source src="/assets/img/old-movies-are-garbage-720.avc.mp4" type="video/mp4">
+>video>
+
+**DIRT IN MY EYE**
+
+There is a lot to be thankful for a career on the digital edge of the motion picture industry.Providing for a fun and fruitful living while getting paid to learn, the revolution was above all very interesting times. 
+
+Chief among the benefits of the sharp and painful transition from film to digital was the massive reduction in physical product manufacturing, distribution and disposal of assets. Thanks to new technology, the movie business is polluting the planet to a slower and lesser degree now than before. 
+
+One of the few projects remembered from grade school was a Grade 4 report on pollution. Relying solely on magazine-type sources for pictures to cut out and glue paste, I easily found more than enough photographs of smoke spewing stacks, poisoned rivers and ruined landscapes to make the grade. The research and images burned my eyes, though; turned my stomach; confused my mind. I couldn't believe mankind could do such a thing to the same planet we eat and breathe from. I swore (durn) I was never in my life going to be a polluter. 
+
+**SMUG CONTROL DEVICE** 
+
+Nevertheless, my first big job was at a smoke-chugging chemical plant – complete with tailing ponds of acidic waste as large as a small city. My resistance to harming the earth melted away as fast as I could cash away that big paycheck, even after daily corrosive dust coatings and breathing does of deadly chemicals. 
+
+Over time, activist-style "memo to the management" essays earned me a reputation as a writer who could reasonably put words together for the big picture. A short-term assignment was given to create a training program for a key transportation function of the plant. Working with Wang word processors (ha!) and video equipment conveniently needed to be performed in an office. Insulated from the noise, fumes, dust and shift work - this was living life in the fast & fresh lane. 
+
+After the project was completed, a clean body and unwilling brain was dumped back out into the pollution and night shifts. Working with computers seemed to have an interesting future beyond typing. Some day they might be able to do pictures - it is just a matter of time. It wasn't too long before I quit for that cleaner living; large paycheck and rational pension plans be damned (or durned). 
+
+**MOUNT FILLMORE** 
+
+Scrub forward much later in the timeline. I worked for a Deluxe kinda company that once processed and distributed a billion feet of acetate-based film (used for movie release prints) per year. That was billion. There was a competing company with a Technicolor name that produced a similar volume over the same long time frame - mountains of it.
```

Comment:
Are you intentionally trying to avoid pointing directly at Deluxe?  I might also suggest a link here so someone can see what Deluxe is.  Possibly same comment for Technicolor, but I think they're a more recognizable name

Reply:
deluxe - double entendra? will link both.